src/util.f90

```diff
@@ -820,7 +820,7 @@ subroutine generate_bunch_bigaussian_weighted( &
  real(dp), allocatable :: ppcb_positions(:,:)
 
  bch=weight
- sigma_cut=3.0
+ sigma_cut=4.0
```

Comment:
I have already reviewed changes like these. Maybe it's time to move the number outside of the source code, if it changing so often?

Reply:
That is totally bizarre, it should be 4! 4 all over the code. It should have been corrected a long time ago.

Reply:
yes, it's because you didn't rebase your code before doing the PR.
To sum up, you are re-submitting PRs you already sent!
The only change is the one Francesco spotted

Reply:
OK, but if the changed was made to the main version it should have already 4. If you could accept all this corrections I will then rebase the *albz branch* since they are now fully synch